$OpenBSD: patch-GNUmakefile_in,v 1.38 2015/01/10 00:38:39 ajacoutot Exp $

- chunk 1->4 : remove silent build lines
- chunk 5 : append extra -O0 cflags for sparc64 which otherwise result in overoptimization
  by emitting a fdtox %r8, %r7 call, whose second operand isn't even and thus invalid.
(on Source/JavaScriptCore/runtime/NumberPrototype.cpp)
- chunk 6->9 : fix variable quoting to properly install doc in a versionned dir

--- GNUmakefile.in.orig	Wed Jan  7 10:48:04 2015
+++ GNUmakefile.in	Fri Jan  9 18:30:52 2015
@@ -12429,11 +12429,11 @@ am__v_P_0 = false
 am__v_P_1 = :
 AM_V_GEN = $(am__v_GEN_@AM_V@)
 am__v_GEN_ = $(am__v_GEN_@AM_DEFAULT_V@)
-am__v_GEN_0 = @echo "  GEN     " $@;
+am__v_GEN_0 =
 am__v_GEN_1 = 
 AM_V_at = $(am__v_at_@AM_V@)
 am__v_at_ = $(am__v_at_@AM_DEFAULT_V@)
-am__v_at_0 = @
+am__v_at_0 =
 am__v_at_1 = 
 DEFAULT_INCLUDES = -I.@am__isrc@
 depcomp = $(SHELL) $(top_srcdir)/Source/autotools/depcomp
@@ -12443,7 +12443,7 @@ COMPILE = $(CC) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES
 	$(CPPFLAGS) $(AM_CFLAGS) $(CFLAGS)
 AM_V_CC = $(am__v_CC_@AM_V@)
 am__v_CC_ = $(am__v_CC_@AM_DEFAULT_V@)
-am__v_CC_0 = @echo "  CC      " $@;
+am__v_CC_0 =
 am__v_CC_1 = 
 CCLD = $(CC)
 LINK = $(LIBTOOL) $(AM_V_lt) --tag=CC $(AM_LIBTOOLFLAGS) \
@@ -12451,13 +12451,13 @@ LINK = $(LIBTOOL) $(AM_V_lt) --tag=CC $(AM_LIBTOOLFLAG
 	$(AM_LDFLAGS) $(LDFLAGS) -o $@
 AM_V_CCLD = $(am__v_CCLD_@AM_V@)
 am__v_CCLD_ = $(am__v_CCLD_@AM_DEFAULT_V@)
-am__v_CCLD_0 = @echo "  CCLD    " $@;
+am__v_CCLD_0 =
 am__v_CCLD_1 = 
 CXXCOMPILE = $(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) \
 	$(AM_CPPFLAGS) $(CPPFLAGS) $(AM_CXXFLAGS) $(CXXFLAGS)
 AM_V_CXX = $(am__v_CXX_@AM_V@)
 am__v_CXX_ = $(am__v_CXX_@AM_DEFAULT_V@)
-am__v_CXX_0 = @echo "  CXX     " $@;
+am__v_CXX_0 =
 am__v_CXX_1 = 
 CXXLD = $(CXX)
 CXXLINK = $(LIBTOOL) $(AM_V_lt) --tag=CXX $(AM_LIBTOOLFLAGS) \
@@ -12465,7 +12465,7 @@ CXXLINK = $(LIBTOOL) $(AM_V_lt) --tag=CXX $(AM_LIBTOOL
 	$(CXXFLAGS) $(AM_LDFLAGS) $(LDFLAGS) -o $@
 AM_V_CXXLD = $(am__v_CXXLD_@AM_V@)
 am__v_CXXLD_ = $(am__v_CXXLD_@AM_DEFAULT_V@)
-am__v_CXXLD_0 = @echo "  CXXLD   " $@;
+am__v_CXXLD_0 =
 am__v_CXXLD_1 = 
 SOURCES = $(Libraries_WebExtensions_libWebExtensionTest_la_SOURCES) \
 	$(Libraries_WebExtensions_libWebProcessTest_la_SOURCES) \
@@ -72779,7 +72779,7 @@ Source/JavaScriptCore/runtime/libjavascriptcoregtk_@WE
 @am__fastdepCXX_FALSE@	$(AM_V_CXX@am__nodep@)$(LIBTOOL) $(AM_V_lt) --tag=CXX $(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) --mode=compile $(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(libjavascriptcoregtk_@WEBKITGTK_API_MAJOR_VERSION@_@WEBKITGTK_API_MINOR_VERSION@_la_CPPFLAGS) $(CPPFLAGS) $(libjavascriptcoregtk_@WEBKITGTK_API_MAJOR_VERSION@_@WEBKITGTK_API_MINOR_VERSION@_la_CXXFLAGS) $(CXXFLAGS) -c -o Source/JavaScriptCore/runtime/libjavascriptcoregtk_@WEBKITGTK_API_MAJOR_VERSION@_@WEBKITGTK_API_MINOR_VERSION@_la-NumberObject.lo `test -f 'Source/JavaScriptCore/runtime/NumberObject.cpp' || echo '$(srcdir)/'`Source/JavaScriptCore/runtime/NumberObject.cpp
 
 Source/JavaScriptCore/runtime/libjavascriptcoregtk_@WEBKITGTK_API_MAJOR_VERSION@_@WEBKITGTK_API_MINOR_VERSION@_la-NumberPrototype.lo: Source/JavaScriptCore/runtime/NumberPrototype.cpp
-@am__fastdepCXX_TRUE@	$(AM_V_CXX)$(LIBTOOL) $(AM_V_lt) --tag=CXX $(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) --mode=compile $(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(libjavascriptcoregtk_@WEBKITGTK_API_MAJOR_VERSION@_@WEBKITGTK_API_MINOR_VERSION@_la_CPPFLAGS) $(CPPFLAGS) $(libjavascriptcoregtk_@WEBKITGTK_API_MAJOR_VERSION@_@WEBKITGTK_API_MINOR_VERSION@_la_CXXFLAGS) $(CXXFLAGS) -MT Source/JavaScriptCore/runtime/libjavascriptcoregtk_@WEBKITGTK_API_MAJOR_VERSION@_@WEBKITGTK_API_MINOR_VERSION@_la-NumberPrototype.lo -MD -MP -MF Source/JavaScriptCore/runtime/$(DEPDIR)/libjavascriptcoregtk_@WEBKITGTK_API_MAJOR_VERSION@_@WEBKITGTK_API_MINOR_VERSION@_la-NumberPrototype.Tpo -c -o Source/JavaScriptCore/runtime/libjavascriptcoregtk_@WEBKITGTK_API_MAJOR_VERSION@_@WEBKITGTK_API_MINOR_VERSION@_la-NumberPrototype.lo `test -f 'Source/JavaScriptCore/runtime/NumberPrototype.cpp' || echo '$(srcdir)/'`Source/JavaScriptCore/runtime/NumberPrototype.cpp
+@am__fastdepCXX_TRUE@	$(AM_V_CXX)$(LIBTOOL) $(AM_V_lt) --tag=CXX $(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) --mode=compile $(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(libjavascriptcoregtk_@WEBKITGTK_API_MAJOR_VERSION@_@WEBKITGTK_API_MINOR_VERSION@_la_CPPFLAGS) $(CPPFLAGS) $(libjavascriptcoregtk_@WEBKITGTK_API_MAJOR_VERSION@_@WEBKITGTK_API_MINOR_VERSION@_la_CXXFLAGS) $(CXXFLAGS) -O0 -MT Source/JavaScriptCore/runtime/libjavascriptcoregtk_@WEBKITGTK_API_MAJOR_VERSION@_@WEBKITGTK_API_MINOR_VERSION@_la-NumberPrototype.lo -MD -MP -MF Source/JavaScriptCore/runtime/$(DEPDIR)/libjavascriptcoregtk_@WEBKITGTK_API_MAJOR_VERSION@_@WEBKITGTK_API_MINOR_VERSION@_la-NumberPrototype.Tpo -c -o Source/JavaScriptCore/runtime/libjavascriptcoregtk_@WEBKITGTK_API_MAJOR_VERSION@_@WEBKITGTK_API_MINOR_VERSION@_la-NumberPrototype.lo `test -f 'Source/JavaScriptCore/runtime/NumberPrototype.cpp' || echo '$(srcdir)/'`Source/JavaScriptCore/runtime/NumberPrototype.cpp
 @am__fastdepCXX_TRUE@	$(AM_V_at)$(am__mv) Source/JavaScriptCore/runtime/$(DEPDIR)/libjavascriptcoregtk_@WEBKITGTK_API_MAJOR_VERSION@_@WEBKITGTK_API_MINOR_VERSION@_la-NumberPrototype.Tpo Source/JavaScriptCore/runtime/$(DEPDIR)/libjavascriptcoregtk_@WEBKITGTK_API_MAJOR_VERSION@_@WEBKITGTK_API_MINOR_VERSION@_la-NumberPrototype.Plo
 @AMDEP_TRUE@@am__fastdepCXX_FALSE@	$(AM_V_CXX)source='Source/JavaScriptCore/runtime/NumberPrototype.cpp' object='Source/JavaScriptCore/runtime/libjavascriptcoregtk_@WEBKITGTK_API_MAJOR_VERSION@_@WEBKITGTK_API_MINOR_VERSION@_la-NumberPrototype.lo' libtool=yes @AMDEPBACKSLASH@
 @AMDEP_TRUE@@am__fastdepCXX_FALSE@	DEPDIR=$(DEPDIR) $(CXXDEPMODE) $(depcomp) @AMDEPBACKSLASH@
@@ -82350,8 +82350,8 @@ install-data-local:
 @ENABLE_WEBKIT1_TRUE@	then echo 1>&2 'No documentation to install' ; \
 @ENABLE_WEBKIT1_TRUE@	else \
 @ENABLE_WEBKIT1_TRUE@	  DOC_MODULE_VERSION=`cat ./Documentation/webkitgtk/version.xml`; \
-@ENABLE_WEBKIT1_TRUE@	  if test -n "$(DOC_MODULE_VERSION)"; then \
-@ENABLE_WEBKIT1_TRUE@	    installdir="$(DESTDIR)$(HTML_DIR)/webkitgtk-$(DOC_MODULE_VERSION)"; \
+@ENABLE_WEBKIT1_TRUE@	  if test -n "$${DOC_MODULE_VERSION}"; then \
+@ENABLE_WEBKIT1_TRUE@	    installdir="$(DESTDIR)$(HTML_DIR)/webkitgtk-$${DOC_MODULE_VERSION}"; \
 @ENABLE_WEBKIT1_TRUE@	  else \
 @ENABLE_WEBKIT1_TRUE@	    installdir="$(DESTDIR)$(HTML_DIR)/webkitgtk"; \
 @ENABLE_WEBKIT1_TRUE@	  fi; \
@@ -82360,9 +82360,9 @@ install-data-local:
 @ENABLE_WEBKIT1_TRUE@	    echo ' $(INSTALL_DATA) '$$i ; \
 @ENABLE_WEBKIT1_TRUE@	    $(INSTALL_DATA) $$i $${installdir}; \
 @ENABLE_WEBKIT1_TRUE@	  done; \
-@ENABLE_WEBKIT1_TRUE@	  if test -n "$(DOC_MODULE_VERSION)"; then \
+@ENABLE_WEBKIT1_TRUE@	  if test -n "$${DOC_MODULE_VERSION}"; then \
 @ENABLE_WEBKIT1_TRUE@	    mv -f $${installdir}/webkitgtk.devhelp2 \
-@ENABLE_WEBKIT1_TRUE@	      $${installdir}/webkitgtk-$(DOC_MODULE_VERSION).devhelp2; \
+@ENABLE_WEBKIT1_TRUE@	      $${installdir}/webkitgtk-$${DOC_MODULE_VERSION}.devhelp2; \
 @ENABLE_WEBKIT1_TRUE@	  fi; \
 @ENABLE_WEBKIT1_TRUE@	fi
 @ENABLE_WEBKIT2_TRUE@	@installfiles=`echo ./Documentation/webkit2gtk/html/*`; \
@@ -82390,8 +82390,8 @@ install-data-local:
 	then echo 1>&2 'No documentation to install' ; \
 	else \
 	  DOC_MODULE_VERSION=`cat ./Documentation/webkitdomgtk/version.xml`; \
-	  if test -n "$(DOC_MODULE_VERSION)"; then \
-	    installdir="$(DESTDIR)$(HTML_DIR)/webkitdomgtk-$(DOC_MODULE_VERSION)"; \
+	  if test -n "$${DOC_MODULE_VERSION}"; then \
+	    installdir="$(DESTDIR)$(HTML_DIR)/webkitdomgtk-$${DOC_MODULE_VERSION}"; \
 	  else \
 	    installdir="$(DESTDIR)$(HTML_DIR)/webkitdomgtk"; \
 	  fi; \
@@ -82400,9 +82400,9 @@ install-data-local:
 	    echo ' $(INSTALL_DATA) '$$i ; \
 	    $(INSTALL_DATA) $$i $${installdir}; \
 	  done; \
-	  if test -n "$(DOC_MODULE_VERSION)"; then \
+	  if test -n "$${DOC_MODULE_VERSION}"; then \
 	    mv -f $${installdir}/webkitdomgtk.devhelp2 \
-	      $${installdir}/webkitdomgtk-$(DOC_MODULE_VERSION).devhelp2; \
+	      $${installdir}/webkitdomgtk-$${DOC_MODULE_VERSION}.devhelp2; \
 	  fi; \
 	fi
 @ENABLE_GTK_DOC_TRUE@	@$(AM_V_GEN) PKG_CONFIG=$(PKG_CONFIG) $(PYTHON) $(srcdir)/Tools/gtk/generate-gtkdoc --rebase --virtual-root=$${DESTDIR}
